### PR TITLE
Automatic tests uiautomator

### PR DESCRIPTION
**Pull request by jabarros**

### CODE REVIEW

File: androidTest/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ownCloud Android client application
+
+  Copyright (C) 2012  Bartek Przybylski
```
Comment (davivel) [inline androidTest/AndroidManifest.xml]: Not here.

File: androidTest/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  Copyright (C) 2012-2015 ownCloud Inc.
```
Comment (davivel) [inline androidTest/AndroidManifest.xml]: 2016

File: androidTest/java/com/owncloud/android/uiautomator/InitialTest.java [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        mDevice.wait(Until.hasObject(By.pkg(launcherPackage).depth(0)), LAUNCH_TIMEOUT);
+
+        // Launch the app
+        Context context = InstrumentationRegistry.getContext();
+        final Intent intent = context.getPackageManager()
+                .getLaunchIntentForPackage(ANDROID_SETTINGS_PACKAGE);
+        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
+        context.startActivity(intent);
+
+        clickByText("Data usage");
```
Comment (davivel) [inline androidTest/java/com/owncloud/android/uiautomator/InitialTest.java]: Please, set a constant for this.

**Comment by davivel** [comment]
Seen, three small changes, then :+1:

**Comment by jesmrec** [comment]
👍

**Comment by davivel** [comment]
:+1:

**Pull request merged**